Tools/WinMLDashboard/README.md

```diff
@@ -28,10 +28,18 @@ The **Edit/View** button switches from Edit mode to View-only mode and vice vers
 
 ## Converting Models
 
-The **Convert** tab (see snip below) helps convert models from several different frameworks (as listed above) to ONNX format. 
+The **Convert** tab (see snip below) helps convert models from several different frameworks (as listed above) to ONNX format by [WinMLTools](https://pypi.org/project/winmltools/). Also you can select the ONNX version to convert to and select to quantize on which OS version. 
```

Comment:
ONNX version to convert to and apply Quantization.

Reply:
changed